<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>411</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnCreateAccount">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:15px;

padding : 0 4px;
background: rgba(0, 0, 160, 1);
border -radius:15px;
color: rgb(255, 255, 255);
font: 24px;
}
QPushButton:hover:!pressed

{
background:#57cc99;
}

QPushButton:pressed
{
              font-size: 19px;
              transition: 0.9s; 
}
</string>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f2f2f2;
border: none;
border-radius: 5px;
padding: 8px;
font-size: 14px;
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEmail">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f2f2f2;
border: none;
border-radius: 5px;
padding: 8px;
font-size: 14px;
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePassword">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f2f2f2;
border: none;
border-radius: 5px;
padding: 8px;
font-size: 14px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
color:rgba(0, 0, 160, 1);
backround-color:white ;</string>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
color:rgba(0, 0, 160, 1);</string>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(0, 0, 160, 1);
font-size: 15px;</string>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 35px;
color:rgba(0, 0, 160, 1);
backround-color:white;</string>
    </property>
    <property name="text">
     <string>Create account!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
